jquery.menu-aim.js

```diff
@@ -1,3 +1,16 @@
+/** @preserve
+ * menu-aim is a jQuery plugin for dropdown menus that can differentiate
+ * between a user trying hover over a dropdown item vs trying to navigate into
+ * a submenu's contents.
+ * @see https://github.com/kamens/jQuery-menu-aim/
+ * @version 1.1 (modified)
+ * @copyright 2014 by Ben Kamens
+ * @license MIT http://en.wikipedia.org/wiki/MIT_License
+ * @modified by Clemens Fiedler, Virtual Identity AG, 2015:
+ * + added noDelayClass option to disable delay on first mouseenter for multi-hierarchy menus
+ * + added submenuDirection == 'both' option for bi-directional multi-hierarchy menus.
+ * + changes are marked with: vi adaption and vi adaption end
+ */
 /**
```

Comment:
the following comment is duplicated:
"menu-aim is a jQuery plugin for dropdown menus that can differentiate.."
just merge the two comment blocks together by deleting lines 13-17.